project/build/dev/d8/themes/custom/patternlab/source/_meta/_00-head.twig

```diff
@@ -15,6 +15,7 @@
   <link rel="stylesheet" href="../../../../dest/style.css">
   <!-- endinject -->
   <link rel="stylesheet" href="../../../../source/styleguide/custom-styleguide-specific.css">
-
+  <script src="https://use.typekit.net/jhz5ozy.js"></script>
```

Comment:
If the face is not brought over, then the hard-coding of the font-weight: bold; in Twig will simply use the browser's faked version. Can you show me a screenshot of the TypeKit configuration for this bundle to verify it's only one face?